covariance/covarianceXsec.h

```diff
@@ -171,6 +195,7 @@ class covarianceXsec : public covarianceBase {
     double *xsec_param_prior_a;
 
     // Contains the parameter names
+	// ETA - don't think we need this anymore tbh
     std::vector<std::string> xsec_param_names;
```

Comment:
We should use parameters name from CovBase

Reply:
done